Documentation/Requirements/Project Summary.md

```diff
@@ -1,15 +1,2 @@
 # Project Summary
-Home Together is a non-profit hub for all things homesharing.
-
-Home Together does not offer a particular service, but rather aims to maximize the number of people and services available in one place. Users can use the website to browse local homesharing services and find people in a particular geographical location, and filter these results by their specific needs.
-
-General features include:
- - Search people and services that match an entered set of filters 
-    - e.g: A user could search by working status, location, pets, and budget. 
-
- - Anonymous and secure sign up.
-    - User enters information on site to verify identity but it is hidden from other users. If user wishes to share personal information with another member they can do so via the site message system. 
-
- - Private no-risk messaging within the website, with email notifications. 
-
- - Rating system for both home sharers and listed services
\ No newline at end of file
+HomeTogether is the non-profit hub for all things home sharing. It aims to not just offer a platform for users to connect but to bring together everything related to home sharing. Users can browse local services such as cleaning and landscaping, look for potential housemates, connect with local classes and seminars, seek out legal advice, and more. Users and businesses will be verified by site admins and User information is hidden from other users unless they explicitly choose to share it. HomeTogether is aimed at everyone from the long retired to the freshly graduated. Businesses will be able to list their services via the classified system for a small fee which helps cover site costs. Not-for-profit organizations and events will be able to list for free. When looking for a housemate users will be able to filter by things such as working status, location, pets and, budget to assist people finding the perfect fit. Users will be able to communicate via a secure on site via a messaging system with e-mail notifications. If users choose to do so they can also share their private details via the same messaging service. Members of the site will also be able to rate and report other users and businesses.
```

Comment:
> Users will be able to communicate via a secure on site via a messaging system with e-mail notifications.

I think there's some extra words in this sentence, maybe the second "via a" shouldn't be there.

Otherwise I think this looks good, though the paragraph makes it a pretty chonky boi. Maybe split these into a few subcategories, one for each user group - home sharer, business, and admin.

Reply:
Yeah that sentence is pretty rough, will re-work.